_about/notes/0.6.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Istio 0.6
+
+order: 95
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Custom Envoy Config**. Pilot now supports ferrying custom Envoy config to the
+proxy. TBD: doc link?
```

Comment:
I'm not aware about this feature. cc @costinm

Reply:
This item refers to this commit:

commit e7eff5d51eda4fd76df914ceb7ae1e350db1f23b
Author: Shriram Rajagopalan <rshriram@users.noreply.github.com>
Date:   Tue Jan 30 22:08:08 2018 -0500

    Pilot Webhooks for adding custom envoy config (#2829)

Reply:
hmm should we put it in readme if no one else knows how to use it?

Reply:
This is how to use it.
https://github.com/mandarjog/istioluawebhook

Reply:
I'll add that link, but we should move this content to istio.io...